runtime/doc/tabpage.txt

```diff
@@ -61,16 +61,25 @@ In the GUI tab pages line you can use the right mouse button to open menu.
 :[count]tabe[dit]				*:tabe* *:tabedit* *:tabnew*
 :[count]tabnew
 		Open a new tab page with an empty window, after the current
-		tab page.  For [count] see |:tab| below.
+		tab page.  If [count] is given the new tab page appears after 
+		the tabpage [count] otherwise the new tab page will appear
```

Comment:
@fmoralesc (small reminder about this, just so you don't forget to address it, or at least respond to it)